flake.crane.nix

```diff
@@ -269,6 +269,13 @@ rec {
     buildPhaseCargoCommand = "patchShebangs ./scripts ; ./scripts/reconnect-test.sh";
   });
 
+  cliTestUpgrade = craneLib.mkCargoDerivation (commonCliTestArgs // {
```

Comment:
You need to add this to one more place in this file if you want it run. And then to `./script/test-ci-all.sh` to trigger from the CI.

Reply:
Where was the other place in the file?  I didn't see that for the other tests

Reply:
```
          cli-test = {
            reconnect = craneBuildNative.cliTestReconnect;
            latency = craneBuildNative.cliTestLatency;
            cli = craneBuildNative.cliTestCli;
            rust-tests = craneBuildNative.cliRustTests;
            always-fail = craneBuildNative.cliTestAlwaysFail;
          };
```